x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/eks_form.tsx

```diff
@@ -82,26 +109,38 @@ const EksForm = ({ onChange, inputs }: Props) => {
   );
 
   return (
-    <EuiDescribedFormGroup title={eksFormTitle} description={eksFormDescription}>
-      {eksVars.map((field) => (
-        <EuiFormRow
-          key={field.id}
-          label={field.label}
-          labelAppend={
-            <EuiText size="xs" color="subdued">
-              <FormattedMessage
-                id="xpack.csp.createPackagePolicy.eksIntegrationSettingsSection.optionalField"
-                defaultMessage="Optional"
-              />
-            </EuiText>
-          }
-        >
-          <EuiFieldText
-            value={values[field.id]}
-            onChange={(event) => onChange(field.id, event.target.value)}
-          />
-        </EuiFormRow>
-      ))}
-    </EuiDescribedFormGroup>
+    <div>
+      <EuiDescribedFormGroup title={eksFormTitle} description={eksFormDescription}>
```

Comment:
this component isn't really suited for the type of content we're showing which makes the text appear a bit crowded. 

in future iterations we might replace it so it looks better